en/_posts/2015-02-07-how-to-use-S3QL-with-runabove.markdown

```diff
@@ -0,0 +1,79 @@
+---
+layout: post
+title: "How to use S3QL with Runabove?"
+categories: Object-Storage
+lang: en
+author: pbtrung
+---
+
+S3QL is a file system that can be mounted locally and stores its data using cloud storage providers like Google Storage, Amazon S3, or OpenStack Swift. Some nice features are transparent compression, encryption, data de-duplication, and snapshotting, which makes it particularly appropriate for backup purposes. More information on the file system can be found on its author' [website](http://www.rath.org/s3ql-docs).
+
+Since Openstack Swift is one of available backends of S3QL, we can use S3QL to save your data on Runabove object storage.
+
+# 1. Set up
+
+## Install S3QL 
+
+The recommended version is 2.x or newer. Guides to install precompiled packages for popular Linux distributions are available on [S3QL wiki](https://bitbucket.org/nikratio/s3ql/wiki/Installation).
```

Comment:
Can you inline documentation for Debian / Ubuntu based distribution ? This will help a lot people people if they can find all essentials help in a single place for the most common setup.